### PR TITLE
Fixes for primary upgrade

### PR DESCRIPTION
**Pull request by jchampio**
Follow-ups to #73. See commit messages.

### CODE REVIEW

File: Gopkg.lock [hunk 1]
```diff
@@ -54,6 +54,29 @@
   revision = "76626ae9c91c4f2a10f34cad8ce83ea42c93bb75"
   version = "v1.0"
 
+[[projects]]
```
Comment (jmcatamney) [inline Gopkg.lock]: gpupgrade doesn't use `jackc/pgx` or `jmoiron/sqlx` directly anymore and they're not in the `Gopkg.toml` file, so should they be in the gpupgrade lock file at all or only in the gp-common-go-libs lock file?

Comment (jchampio) [reply Gopkg.lock]: I think lock files are supposed to be transitively complete -- all the dependencies of the dependencies of the dependencies come in.

**Comment by jchampio** [comment]
I added one more commit, which reenables `--progress` mode for pg_upgrade. (You'll need to update to a recent GPDB master tip.)

With that, `make installcheck` passes with GPDB 6 on my machine.

File: Gopkg.lock (hunk 1 above)
Comment (jmcatamney) [reply Gopkg.lock]: In that case, looks good to me.

**Comment by jchampio** [comment]
Thanks for the review!

**Pull request merged**